packages/prediction-app/src/hooks/usePlotDataForEvaluations.ts

```diff
@@ -87,7 +92,7 @@ export const usePlotDataForEvaluations = (
 
     /* Try to find a query in the cache that has data for all selected orgunits */
     const getInitialData = useCallback(
-        (evaluationId: number) => {
+        (evaluationId: number): PlotDataRequestResult | undefined => {
```

Comment:
Sorry... This should just contain some type improvements/cleanup and formatting.